websites/N/Night.design/presence.ts

```diff
@@ -0,0 +1,86 @@
+const presence = new Presence({
+    clientId: "1031730262092042321"
+})
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "logo",
+	}
+
+    presenceData.startTimestamp = Math.floor(Date.now() / 1000);
+    presenceData.buttons = [
+        {
+            label: "View Website",
+            url: "https://night.design/"
+        },
+        {
+            label: "Actually Page",
+            url: document.location.href
+        }
+    ]
+
+    let domain = document.location.pathname.slice(3);
```

Comment:
`domain` is a static variable so should use `const` instead

Reply:
Updated.